common/service/mctp/mctp_i3c.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mctp.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <zephyr.h>
+#include <sys/crc.h>
+#include <logging/log.h>
+#include "libutil.h"
+#include "hal_i3c.h"
+
+LOG_MODULE_REGISTER(mctp_i3c);
+
+#ifndef MCTP_I3C_PEC_ENABLE
+#define MCTP_I3C_PEC_ENABLE 0
+#endif
+
+static uint16_t mctp_i3c_read_smq(void *mctp_p, uint8_t *buf, uint32_t len,
+				  mctp_ext_params *extra_data)
+{
+	CHECK_NULL_ARG_WITH_RETURN(mctp_p, MCTP_ERROR);
+	CHECK_NULL_ARG_WITH_RETURN(buf, MCTP_ERROR);
+
+	return 0;
+}
```

Comment:
Is this just a function stub at this point? It doesn't look like it does anything.

If that's the case can we return some type of error and possibly print a message so it's more obvious it doesn't currently do anything / isn't implimented.

Reply:
Yes, it is for now.
Added LOG_INF message and leave comment in the function.
We will add the code in Rx commit.

Please review it again. Thank you.